jrcc-document-access-libs/src/main/java/ca/bc/gov/open/jrccaccess/libs/TransactionInfo.java

```diff
@@ -56,10 +58,12 @@ public String getFileName() {
 	public LocalDateTime getReceivedOn() {
 		return receivedOn;
 	}
+
+	public String getUUIDStr(){return this.uuid.toString();}
 	
```

Comment:
I would rather return the UUID and let the client code convert to appropriate string format

Reply:
sure

Reply:
question: if it is all on one line, can we write it without the `{`? might look a bit cleaner :) 
```suggestion
public String getUUIDStr() return this.uuid.toString();
```

Reply:
for Java, that won't compile.

Reply:
yes of course, my bad. I was thinking if statements, not function definitions